### PR TITLE
Added vuetiful-datatable

### PR DESCRIPTION
**Pull request by grakic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,8 @@
   - [Enterprise Usage](#enterprise-usage)
 - [Components & Libraries](#components--libraries)
   - [UI Components](#ui-components)
-    - [Table](#table)
+    - [
```
Comment (kazupon) [inline README.md]: Why do you edit?

File: README.md [hunk 2]
```diff
@@ -240,7 +241,6 @@
 - [Complete Vue.js Application Tutorial - Creating a Simple Budgeting App with Vue](http://matthiashager.com/complete-vuejs-application-tutorial) by [@matthiaswh](https://github.com/matthiaswh)
 - [Vue.js Tutorial: A Prerendered, SEO-Friendly Example](https://snipcart.com/blog/vuejs-tutorial-seo-example)
 - [Vue.js Introduction For People Who Know Just Enough jQuery To Get By](https://medium.com/@mattrothenberg/vue-js-introduction-for-people-who-know-just-enough-jquery-to-get-by-eab5aa193d77)
-- [Fetching Data from a Third-Party API with Vue.js and Axios](https://www.sitepoint.com/fetching-data-third-party-api-vue-axios/)
```
Comment (kazupon) [inline README.md]: Why do you remove?

File: README.md [hunk 3]
```diff
@@ -825,7 +826,6 @@ Tooltips / popovers
  - [vue-clickaway](https://github.com/simplesmiler/vue-clickaway) - Reusable clickaway directive for reusable Vue.js components.
  - [vue-scrollfire](https://github.com/vue-comps/vue-scrollfire) - Fires an event on a specific scroll position.
  - [vue-resize-directive](https://github.com/David-Desmaisons/Vue.resize) - Vue directive to detect resize events with deboucing and throttling capacity.
- - [v-click-outside](https://github.com/ndelvalle/v-click-outside) - Vue directive to react on clicks outside of an element without stopping the event propagation.
```
Comment (kazupon) [inline README.md]: Why do you remove here ?

**Comment by grakic** [comment]
Fixed, my editor went crazy :goat:

**Comment by kazupon** [comment]
thanks!

**Pull request merged**